### PR TITLE
Eperez enum letter proofing

### PR DESCRIPTION
**Pull request by enriquepablo**
Keep messages to the user in the letter proofing app in an enum

### CODE REVIEW

**Comment by enriquepablo** [comment]
codecov failing in previous pr

**Comment by enriquepablo** [comment]
ignore codecov

File: src/eduid_webapp/letter_proofing/views.py [hunk 1]
```diff
@@ -55,7 +56,7 @@ def proofing(user, nin):
         current_app.logger.info('A letter has already been sent to the user. ')
         current_app.logger.debug('Proofing state: {}'.format(proofing_state.to_dict()))
         result = check_state(proofing_state)
-        if not result['letter_expired']:
+        if 'letter_expired' not in result or not result['letter_expired']:
```
Comment (fredrikt) [inline src/eduid_webapp/letter_proofing/views.py:59]: This if-clause hurts my poor brain a bit. Can you rewrite it to

`
        if 'letter_expired' not in result:
            # error message
            return result
        if not result['letter_expired']:
            return result
        # XXX Are we sure that the user wants to send a new letter?
        current_app.logger.info('The letter has expired. Sending a new one...')
`

thanks :)

Comment (enriquepablo) [reply src/eduid_webapp/letter_proofing/views.py]: done! :)

**Pull request merged**